drafts/versions/02/draft-ietf-ippm-ioam-data-02.txt

```diff
@@ -1070,15 +1070,64 @@ Internet-Draft           In-situ OAM Data Fields           February 2018
 
    IOAM proof of transit option header:
 
-    0 1 2 3 4 5 6 7
-   +-+-+-+-+-+-+-+-+
-   |IOAM POT Type|P|
-   +-+-+-+-+-+-+-+-+
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |IOAM POT Type  | IOAM POT flags|           Reserved            |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+   IOAM proof of transit option data MUST be 4-octet aligned.:
 
-   IOAM proof of transit option data MUST be 4-octet aligned:
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |       POT Option data field determined by IOAM-POT-Type       ~
```

Comment:
I assume '~' at the end is a typo?
Also wondering why this is indented one more space than a few lines above.